index.html

```diff
@@ -0,0 +1,176 @@
+                    Digital-Payement,
+                    cryptocurrency,
+                </p>
+            </div>
+            <h2 class="headline-date">2022-02-08 (MON) - 12 (THU)</h2>
+            <p class="headline-location">@ ASSTU center of inovation and digital transformation</p>
+        </section>
+        <section class="main-program-section">
+            <div class="title-container">
+                <h2 class="title">Main Program</h2>
+                <div class="divider"></div>
+            </div>
+
+            <div class="main-program" id="main-program">
+                <div class="program">
+                    <img src="./image/devops.png" alt="devops">
+                    <h3 class="program-title">DevOps</h3>
+                    <p class="program-description">
+
+                        We will share our experiance on DevOps and Agile software development methodology
+                    </p>
+                </div>
+                <div class="program">
+                    <img src="./image/ai.jpg" alt="machine learning">
+                    <h3 class="program-title">Machine Learning</h3>
+                    <p class="program-description">
+                        We will share our experiance on DevOps and Agile software development methodology
+                    </p>
+                </div>
+                <div class="program">
+                    <img src="./image/forum.png" alt="forum">
+                    <h3 class="program-title">Forum</h3>
+                    <p class="program-description">
+                        We will share our experiance on DevOps and Agile software development methodology
+                    </p>
+                </div>
+                <div class="program">
+                    <img src="./image/workshop.png" alt="workshop">
+                    <h3 class="program-title">Workshop</h3>
+                    <p class="program-description">
+                        You can Try inovate your own technology using open source license rather than just watching.
+                    </p>
+                </div>
+                <div class="program">
+                    <img src="./image/data.png" alt="ata Scinces">
+                    <h3 class="program-title">Data Scinces</h3>
+                    <p class="program-description">
+                        Get opportunities to network with CC affiliate around the world, also after the submit.
+                    </p>
+                </div>
+            </div>
+            <button type="button" class="btn btn-lg btn-join">Join our association 2022 </button>
+            <a href="#" class="desk-see-whole">SEE THE WHOLE PROGRAM</a>
+        </section>
+        <section class="future-product" id="product">
+            <div class="title-container">
+                <h2 class="title">Featured Product</h2>
+                <div class="divider"></div>
```

Comment:
The `Featured Product` should have to match the given design. At the back of the image there should have to be a square black and white squares as show below in the design. Also implement the `More` button in mobile version as of the design when it's clicked it will unhide the rest of the featured products. And the text is off to the right please give it some padding.

| Your Design | Design provided |
|-------------|---------------|
| <img width="360" alt="image" src="https://user-images.githubusercontent.com/34578631/181301389-d45610bf-b0e8-41db-9666-817bad76985a.png"> | <img width="555" alt="image" src="https://user-images.githubusercontent.com/34578631/181301459-a4f8ab25-7e57-4e22-b4ec-a9de551799c2.png"> |

Reply:
@Mre55 Thank you for your constructive feedback. According to the requirement **More** button is an Additional requirement. 
![2022-07-27_23-19-07](https://user-images.githubusercontent.com/30173722/181364721-1ebe8efc-88b2-4092-80b4-068c9ef20ec1.png)

Reply:
@melashu, you're right and you can take it as OPTIONAL.